### PR TITLE
fix(chore/pr template): update git conventions and PR template

### PR DESCRIPTION
**Pull request by gael-boyenval**
Change PR template

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,42 +1,19 @@
...
+## I have read [the contributing guidelines](https://mozaic.adeo.cloud/contributing)
 
+- [ ] Yes
+- [ ] No
 
 ## Does this PR introduce a breaking change?
 
+read [What is a breaking change ?]()
+
 - [ ] Yes
```
Comment (tiloyi) [inline .github/PULL_REQUEST_TEMPLATE.md]: Maybe you can link this page **/Contributing/GitConventions/.**
There is a small definition of what it is, in the "Naming convention" section.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,42 +1,19 @@
...
 - [ ] No
 
+## Describe the changes
 
-<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->
+<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
 
+Github issue number or Jira issue URL: N/A
```
Comment (tiagoux) [inline .github/PULL_REQUEST_TEMPLATE.md]: not everybody uses JIRA. We could also add Issue # Jira or Issue #

File: src/pages/Contributing/GitConventions/index.md [hunk 3]
```diff
@@ -13,10 +13,8 @@ git checkout -b type-id-description
 
 > MOZAIC Team use **JIRA as their project management tool** <br> BU's use **Github issues to manage their tasks into MOZAIC**
```
Comment (tiagoux) [inline src/pages/Contributing/GitConventions/index.md]: MOZAIC's team is using Jira...

File: src/pages/Contributing/GitConventions/index.md [hunk 4]
```diff
@@ -28,49 +26,47 @@ Each commit message consists of a `type`, an optional `scope` and a `description
 feat(component/button): add a warning color theme
 ```
 
-> Commit message's lines **cannot be longer than 140 characters**! <br> This leads to **more readable messages** that are easy to follow when looking through the project history. 
+> Commit message's lines **cannot be longer than 140 characters**! <br> This leads to **more readable messages** that are easy to follow when looking through the project history.
```
Comment (tiagoux) [inline src/pages/Contributing/GitConventions/index.md]: This leads to **easier to read messages** throughout the project history.

File: src/pages/Contributing/GitConventions/index.md [hunk 5]
```diff
@@ -28,49 +26,47 @@ Each commit message consists of a `type`, an optional `scope` and a `description
...
 
-We are **automating the releases version numbers (SEMVER) and the generation of the changelogs files by parsing the commit messages**. 
+We are **automating the releases version numbers (SEMVER) and the generation of the changelogs files by parsing the commit messages**.
 
 That's why it's very important to follow the conventional commit conventions, and to be very thoughtful about the description you provide
```
Comment (tiagoux) [inline src/pages/Contributing/GitConventions/index.md]: link for conventional commit conventions? 

and to be very thoughtful -> and to be as clear as possible on the description

File: src/pages/Contributing/GitConventions/index.md [hunk 6]
```diff
@@ -28,49 +26,47 @@ Each commit message consists of a `type`, an optional `scope` and a `description
...
-    * **tools/xxx** : any dev tool delivered to the end user like linters, postcss configs, tokens customization scripts etc
-    * **chore/xxx** : anything related to CI, devtools, scripts for mozaic developers
+  - **website/xxx** : everything that has to do with the mosaic website UI, like menus, tabs nav, Gatsby etc...
+  - **onboard/xxx** : documentation only like getting started, contributing sections and information pages.
+  - **patterns/xxx** : anything that is related to visual deliverables : code / design / doc / tokens / icons
+  - **tools/xxx** : any dev tool delivered to the end user like linters, postcss configs, tokens customization scripts etc
+  - **chore/xxx** : anything related to CI, devtools, scripts for mozaic developers
 
-    > Note: **xxx points to more precise scope**. For example : `patterns/buttons`, `website/menu`
+  > Note: **xxx points to more precise scope**. For example : `patterns/buttons`, `website/menu`
```
Comment (tiagoux) [inline src/pages/Contributing/GitConventions/index.md]: xxx aims to have a more precise scope. For example ...

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (gael-boyenval) [reply .github/PULL_REQUEST_TEMPLATE.md]: It's why we use GitHub issue or Jira issue. What would you like us to write ?

File: src/pages/Contributing/GitConventions/index.md [hunk 7]
```diff
@@ -11,12 +11,10 @@ Each branch name consists of a `type` that matches the main commit one, an optio
 git checkout -b type-id-description
 ```
 
-> MOZAIC Team use **JIRA as their project management tool** <br> BU's use **Github issues to manage their tasks into MOZAIC**
-
+> MOZAIC's Team is using **JIRA as their project management tool** <br> BU's use **Github issues to manage their tasks into MOZAIC**
```
Comment (tiagoux) [inline src/pages/Contributing/GitConventions/index.md]: ~BU's~ - contributors

File: src/pages/Contributing/GitConventions/index.md [hunk 8]
```diff
@@ -28,49 +26,47 @@ Each commit message consists of a `type`, an optional `scope` and a `description
...
-    * **tools/xxx** : any dev tool delivered to the end user like linters, postcss configs, tokens customization scripts etc
-    * **chore/xxx** : anything related to CI, devtools, scripts for mozaic developers
+  - **website/xxx** : everything that has to do with the mosaic website UI, like menus, tabs nav, Gatsby etc...
+  - **onboard/xxx** : documentation only like getting started, contributing sections and information pages.
+  - **patterns/xxx** : anything that is related to visual deliverables : code / design / doc / tokens / icons
+  - **tools/xxx** : any dev tool delivered to the end user like linters, postcss configs, tokens customization scripts etc
+  - **chore/xxx** : anything related to CI, devtools, scripts for mozaic developers
 
-    > Note: **xxx points to more precise scope**. For example : `patterns/buttons`, `website/menu`
+  > Note: **xxx refer to more precise scope**. For example : `patterns/buttons`, `website/menu`
```
Comment (tiagoux) [inline src/pages/Contributing/GitConventions/index.md]: refers

**Pull request merged**